Userland/Applications/Mail/MailWidget.cpp

```diff
@@ -368,10 +368,17 @@ void MailWidget::selected_mailbox()
 
         auto& from_iterator_value = from_iterator->get<1>().value();
         auto from_index = from_iterator_value.find("From:"sv);
-        VERIFY(from_index.has_value());
-        auto potential_from = from_iterator_value.substring(from_index.value());
-        auto from_parts = potential_from.split_limit(':', 2);
-        auto from = parse_and_unfold(from_parts.last());
+        if (!from_index.has_value())
+            from_index = from_iterator_value.find("from:"sv);
+        DeprecatedString from;
+        if (from_index.has_value()) {
+            auto potential_from = from_iterator_value.substring(from_index.value());
+            auto from_parts = potential_from.split_limit(':', 2);
+            from = parse_and_unfold(from_parts.last());
+        }
+
+        if (from.is_empty())
+            from = "(Unknown sender)";
```

Comment:
What do the relevant RFCs say about the case sensitivity of email headers? Should we do a case-insensitive match instead?

Reply:
[RFC-2076](https://www.rfc-editor.org/rfc/rfc2076#section-3.4) sums it up nicely, the actual header should be: `From:`, but in practice, my inbox of ~25k messages contained one single message where that header was `from:`, and no messages where the `Subject:` field deviated from specifications.
I don't mind doing a case-insensitive match, but that falls beyond the scope of this PR, I don't think we have a `DeprecatedString::find()` overload that takes a `CaseSensitivity` input.